src/components/table/mobile/table_sort_mobile.js

```diff
@@ -65,14 +63,29 @@ export class EuiTableSortMobile extends Component {
         ownFocus
         button={mobileSortButton}
         isOpen={this.state.isPopoverOpen}
-        closePopover={this.closePopover.bind(this)}
-        anchorPosition={anchorPosition || "downRight"}
```

Comment:
Any particular reason not to allow this prop to be passed optionally?

Reply:
I had a good reason at some point 😆will add back